gr-azure-software-radio/python/integration_default_credentials.py

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+#
+# Copyright (c) Microsoft Corporation.
+# Licensed under the GNU General Public License v3.0 or later.
+# See License.txt in the project root for license information.
+#
+
+from gnuradio import gr_unittest
+import pytest
+import azure_software_radio.default_credentials
+
+
+class integration_default_credentials(gr_unittest.TestCase):
+
+    def test_001_environment_only(self):
```

Comment:
nit: Do we want to standardize on numbering tests or no? I feel like we should either number them all or none. I lean towards none just in case we add one in the middle of a file, it either looks messy since we're now out of order, or we waste a bunch of time renumbering test cases, which takes away from time spent improving code test coverage.

Reply:
removed numbering